### ISSUE TITLE
size of brief.sqlite-wal rises and is not deleted when closing firefox

### ISSUE THREAD

**Issue by jay1109** [tanriol/digest#78]
Firefox 23.0.1 on Windows XP SP3

**Comment by tanriol** [comment]
How much does it reach? AFAIK it may be up to 32 Mb... does it become larger than that?

**Comment by jay1109** [comment]
brief.sqlite is 50MB
brief.sqlite-wal is over 400MB

I reinstalled the extension, but it seams that it does not solve the issue
brief.sqlite 500K
brief.sqlite-wal 23MB

**Comment by tanriol** [comment]
Hmmm... seems to be a problem. Reinstallation is not expected to help here. I'll look into it.

**Comment by jay1109** [comment]
thank you.

**Comment by kid23** [comment]
Yes. I have also come across this problem. When I used the brief for a long time, the file once could be more than 1GB. 
No idea. I used sqlite3 command line to reduce -wal file manually.

sqlite3 brief.sqlite
VACUUM;
.quit

I hope @Tanriol could solve this problem:)
